From: Miao Wang <shankerwangmiao@gmail.com>
Date: Sun, 4 Jan 2026 17:31:03 +0800
Subject: Add necessary definations to the Kbuild file

---
 src/Kbuild | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Kbuild b/src/Kbuild
index 7b16f0f..cf7b741 100644
--- a/src/Kbuild
+++ b/src/Kbuild
@@ -1,3 +1,4 @@
 yt6801-objs := fuxi-gmac-common.o fuxi-gmac-desc.o fuxi-gmac-ethtool.o fuxi-gmac-hw.o fuxi-gmac-net.o fuxi-gmac-pci.o fuxi-gmac-phy.o fuxi-efuse.o  fuxi-gmac-ioctl.o
 obj-m += yt6801.o
 ccflags-y += -DFXGMAC_ASPM_ENABLED -DFXGMAC_EPHY_LOOPBACK_DETECT_ENABLED
+ccflags-y += -DFXGMAC_INT_MODERATION_ENABLED=1 -DINT_MOD_IN_US=200
